contracts/USM.sol

```diff
@@ -29,13 +27,28 @@ contract USM is ERC20 {
         ethPool = 0;
     }
 
+    /**
+     * @notice Multiplies x and y, assuming they are both fixed point with 27 digits.
+     */
+    function muld(uint256 x, uint256 y) internal pure returns (uint256) {
+        return x.mul(y).div(UNIT);
+    }
+
+    /**
+     * @notice Divides x between y, assuming they are both fixed point with 27 digits.
```

Comment:
Change to: `@notice Divides x by y, assuming they are both fixed point with 27 digits.`

Reply:
Me English no good :P